api/serializers.py

```diff
@@ -96,6 +96,52 @@ class Meta:
         )
 
 
+class GeneralCatalogExecutorCardSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+    price = serializers.SerializerMethodField()
+    portfolio = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = (
+            'id',
+            'full_name',
+            'is_client',
+            'is_photographer',
+            'is_video_operator',
+            'about_me',
+            'price',
+            'portfolio',
+        )
+
+    def get_full_name(self, obj):
+        return f'{obj.first_name} {obj.last_name}'
+
+    def get_price(self, obj):
+        services = obj.services.all()
+        if services:
+            lower_price = services.order_by('cost_service')[0].cost_service
+            return lower_price
+
+    def get_portfolio(self, obj):
+        all_media = obj.portfolio.all()
```

Comment:
Мне кажется можно сразу добавить сортировку  
```suggestion
        all_media = obj.portfolio.all().order_by('media_file__media_type')
```